README.md

```diff
@@ -57,7 +57,7 @@ pnpm run build:preview
 npx http-server ./dist
 ```
 
-Note that changes to config or build scripts will require re-running `pnpm start`.
+Note that changes to `src` or `build` scripts will require re-running `pnpm start`.
```

Comment:
Question: isn't the point to NOT have to rerun `pnpm start` when changing a `src` file?

Reply:
not the point of this PR -- `pnpm start` doesn't have any "watch mode" implemented -- and I'm not sure how to implement it -- maybe I could switch everything to rollup plugins? but idk -- problem for another time